subsys/bluetooth/mesh/adv.c

```diff
@@ -241,6 +247,34 @@ struct bt_mesh_adv *bt_mesh_adv_get_by_tag(enum bt_mesh_adv_tag_bit tags, k_time
 	return bt_mesh_adv_get(timeout);
 }
 
+bool bt_mesh_adv_is_empty_by_tag(enum bt_mesh_adv_tag_bit tags)
```

Comment:
Can we create a function that just returns a fifo and then either call `k_fifo_is_empty` or `k_fifo_get`? Currently this duplicates logic of `bt_mesh_adv_get_by_tag`. If the latter needs to be fixed, this one can be broken then. Especially considering that we don't have unit tests for this code at all.